### PR TITLE
Remove repetitive/redundant code

### PR DESCRIPTION
**Pull request by eumiro**

### CODE REVIEW

File: tests/test_cmdline.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def test_dump(script_runner):
     ret = script_runner.run('tnefparse', '-d', 'tests/examples/two-files.tnef')
     assert ret.success
     dump = json.loads(ret.stdout)
-    assert sorted(list(dump.keys())) == ['attachments', 'attributes', 'extended_attributes']
+    assert sorted(dump) == ['attachments', 'attributes', 'extended_attributes']
```
Comment (jugmac00) [inline tests/test_cmdline.py:71]: Wow, I learned something new.

Following the Zen of Python, I'd still personally prefer:
```
assert sorted(dump.keys())
```
even when it is not necessary.

File: tnefparse/tnef.py [hunk 2]
```diff
@@ -399,25 +396,20 @@ def to_zip(data, default_name='no-name', deflate=True):
     tozip = {}
     for attachment in tnef.attachments:
         filename = attachment.name or default_name
-        L = len(tozip.get(filename, []))
-        if L > 0:
+        if tozip.get(filename):
+            length = len(tozip.get(filename))
```
Comment (jugmac00) [inline tnefparse/tnef.py:400]: There is no coverage for this line. It would be great to have a test for it.

File: tnefparse/tnef.py [hunk 3]
```diff
@@ -399,25 +396,20 @@ def to_zip(data, default_name='no-name', deflate=True):
...
             # uniqify this file name by adding -<num> before the extension
-            root, ext = os.path.splitext(filename)
-            tozip[filename].append((attachment.data, str("%s-%d%s" % (root, L + 1, ext))))
-        else:
-            tozip[filename] = [(attachment.data, filename)]
+            filename = filename.with_stem(f"{filename.stem}-{length + 1}")
```
Comment (jugmac00) [inline tnefparse/tnef.py:402]: There is no coverage for this line. It would be great to have a test for it.

File: tnefparse/util.py [hunk 4]
```diff
@@ -45,7 +45,7 @@ def systime(byte_arr, offset=0):
         return datetime.utcfromtimestamp((ft - EPOCH_AS_FILETIME) / HUNDREDS_OF_NANOSECONDS)
     except:  # noqa: E722
         microseconds = ft / 10
-        return (datetime(1601, 1, 1) + timedelta(microseconds=microseconds))
+        return datetime(1601, 1, 1) + timedelta(microseconds=microseconds)
```
Comment (jugmac00) [inline tnefparse/util.py:48]: There is no coverage for this line. It would be great to have a test for it.

File: tnefparse/tnef.py (hunk 3 above)
Comment (eumiro) [reply tnefparse/tnef.py:402]: This will be done in #89

File: tnefparse/util.py (hunk 4 above)
Comment (jugmac00) [reply tnefparse/util.py:48]: Seems to be non-trivial to come up with a test case https://github.com/koodaamo/tnefparse/issues/92#issuecomment-735883589

**Comment by jrideout** [comment]
Rebased changes were merged in #103

**Pull request closed without merging**